covalent_dispatcher/_db/models.py

```diff
@@ -182,9 +182,8 @@ class Electron(Base):
     # Name of the file containing standard error generated by the task
     stderr_filename = Column(Text)
 
-    # Name of the file containing execution information generated at
-    # runtime (such as Slurm logs)
-    info_filename = Column(Text)
```

Comment:
@cjao is this just a renaming of info_filename to error_filename ? do they contain effectively the same content?

Reply:
The `info_filename` was never actually used, and the semantics are different. Here, `error_filename` contains errors generated by `Covalent` during a task's execution, not by the executor backend.

Reply:
Sounds good @cjao , if it was indeed used I would have asked to include a data migration as well to update the `error_filename` with the `info_filename` values but all good here!